payment_manual/indico_payment_manual/plugin.py

```diff
@@ -58,3 +68,10 @@ def logo_url(self):
 
     def get_blueprints(self):
         return IndicoPluginBlueprint('payment_manual', __name__)
+
+    def adjust_payment_form_data(self, data):
+        event = data['event']
+        event_settings = data['event_settings']
```

Comment:
is it really worth assigning all of them to variables first? doesn't look too bad with just getting the dict items when calling `replace_placeholders` IMO